tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "./tsconfig.base.json",
   "files": [],
   "typedocOptions": {
-    "packages": [
+    "entryPointStrategy": "packages",
+    "entryPoints": [
```

Comment:
Fixed the breaking change introduced in https://typedoc.org/guides/changelog/#v0.22.0-(2021-09-10).